app/controllers/categories_controller.rb

```diff
@@ -41,7 +41,7 @@ def update
   private
 
   def category_params
-    params.permit(:name, :description, :status)
+    params.permit(:name, :description, :status, :created_at)
```

Comment:
Why we need `:created_at` here? Active Record automatically timestamps create operation.

Reply:
Ok,  I'll remove it

Reply:
@evrusha Done